lib/src/area.dart

```diff
@@ -0,0 +1,130 @@
+import 'dart:math';
+
+import '../helpers.dart';
+import '../meta.dart';
+
+/// Takes one or more features and returns their area in square meters.
```

Comment:
I would replace `Takes one or more features` with `Takes a [GeoJSONObject]`, because that way we have made use of the type annotation for Dart docs.